homeworks/roman.strokan_Strokan/homework_3/z1.js

```diff
@@ -0,0 +1,46 @@
+
+// 1.1
+function func(a) {
+    const b = a * a;
+    // console.log(`${a}^2=${b}`);
+    return b;
+}
+// func(5);
+
+// 1.2
+function arrayCreation(a, b) {
+    const arr = [];
+    let c = 0;
+    let i = 0;
+    do {
+        arr[i] = c;
+        c += a;
+        i++;
+    } while (c <= b);
+    /* for (let j = 0; j < arr.length; j++) {
+        console.log(arr[j]);
+    } */
+    return arr;
```

Comment:
as an advice, avoid using do...while cycle wherever you can, the result may be OK, but in general it's usually used for particular purposes... here any other cycle may be fine